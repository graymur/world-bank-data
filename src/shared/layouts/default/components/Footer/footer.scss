.footer {
    height: 6rem;

    .container {
        border-top: 1px solid #ececec;
        padding-top: 2rem;
    }
}