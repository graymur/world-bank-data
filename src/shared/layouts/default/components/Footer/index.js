import React from 'react';
import {pure} from 'recompose';
import './footer.scss';

const Footer = () => (
	<footer className='footer'>
		<div className='container'>
			{(new Date()).getFullYear()} <a href='https://github.com/graymur/world-bank-data' target='_blank'>graymur/world-bank-data</a>
		</div>
	</footer>
);

export default pure(Footer);
