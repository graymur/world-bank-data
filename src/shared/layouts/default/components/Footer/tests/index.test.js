/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../index';

test('Renders content in wrapper', () => {
	const component = renderer.create(
		<Footer/>
	);

	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
