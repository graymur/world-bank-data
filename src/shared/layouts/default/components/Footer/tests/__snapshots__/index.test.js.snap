// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders content in wrapper 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    2018
     
    <a
      href="https://github.com/graymur/world-bank-data"
      target="_blank"
    >
      graymur/world-bank-data
    </a>
  </div>
</footer>
`;
